multigeiger/multigeiger.ino

```diff
@@ -354,6 +355,27 @@ void setup()
     while (!Serial) {};
   }
 
+  // build revString
+  strcat(revString,software_version);
+  strcat(revString," ");
+  strcat(revString,software_version_date);
```

Comment:
how about using `sprintf`?

you could then use "V%s %s" as format string and also remove the "V" initialiser in line 233.

this also removes the theoretical problem that the code in this PR only works once (which is no real problem, as setup() is only executed once, but still a bit unpretty.

Reply:
ok, sounds better